Clinical-Genomics/preClinVar
# Convert csv files to json submission, with validation against schema

### This PR adds | fixes:
- Convert csv files to json object
- If they are malformed or one of the 2 is missing, return error with error message

**How to prepare for test**:
- [ ] `ssh` to ...
`bash servers/resources/SERVER.scilifelab.se/update-[THIS_TOOL]-stage.sh [THIS-BRANCH-NAME]`

### How to test:
- Try to load only of the 3 files and you should get error (400 error)
- Try to load a malformed file and the endpoint should return which fields are wrong (400 error)
- Load both files (variant and CaseData) and endpoint should return a json submission object - validation OK - response code 200

### Expected outcome:

### Review:

This [version](https://semver.org/) is a:
